libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2885,6 +2899,29 @@ MAV_RESULT GCS_MAVLINK::handle_flight_termination(const mavlink_command_long_t &
     return MAV_RESULT_FAILED;
 }
 
+/*
```

Comment:
Maybe group this with `GCS_MAVLINK::send_high_latency2` ?

Reply:
Sure, I've moved that in